tests/de.json

```diff
@@ -0,0 +1 @@
+{"animals":{"ant":"ant","bat":"bat","cat":"cat"},"colors":{"white":"white","black":"black","yellow":"yellow","pink":"pink"},"winner":"winner"}
```

Comment:
With this formatting, after having run `npm test` the formatting of this file has changed to:

```
{
  "animals": {
    "ant": "ant",
    "bat": "bat",
    "cat": "cat"
  },
  "colors": {
    "white": "white",
    "black": "black",
    "yellow": "yellow",
    "pink": "pink"
  },
  "winner": "winner"
}
```

So maybe commit the file like that?